.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: Python CI
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+  release:
+    types: [released]
+
+jobs:
+  test:
+    name: Python ${{ matrix.python-version }} (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - python-version: "3.9"
+            platform: ubuntu-latest
+            toxenv: py39
+          - python-version: "3.10"
+            platform: ubuntu-latest
+            toxenv: py310
+          - python-version: "3.11"
+            platform: ubuntu-latest
+            toxenv: py311
+          - python-version: "3.12"
+            platform: ubuntu-latest
+            toxenv: py312
+          - python-version: "3.13"
+            platform: ubuntu-latest
+            toxenv: py313
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install tox
+        run: python -m pip install --upgrade tox
+
+      - name: Run lint
+        run: tox -e lint
+
+      - name: Run tests
+        run: tox -e ${{ matrix.toxenv }}
+
+      - name: Run Codecov
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: tox -e codecov
+
+  publish:
+    name: Build & Publish to PyPI
+    if: github.event_name == 'release'
+    needs: test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install tox
+        run: python -m pip install --upgrade tox
+
+      - name: Publish to PyPI
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
These secrets are named `TWINE_*` on grizzly. `PYPI_*` is probably more correct, but we should be consistent by using one or the other.

Reply:
Yeah, let's change it to PYPI_* everywhere.